Sources/SnowManLogic/Game.swift

```diff
@@ -0,0 +1,36 @@
+import Dictionary
+
+public class Game {
+	let dictionary: DictionaryManager
+	let view: View
+
+	var correctGuesses: [String] = []
+	var wrongGuesses: [String] = []
+	var gameWord: String = ""
+
+	public init(dictionary: DictionaryManager, view: View) {
+		self.dictionary = dictionary
+		self.view = view
+	}
+
+	public func getWord() -> String {
+		gameWord = dictionary.getRandomWord()
+		return gameWord
+	}
+
+	public func checkIfGuessingFullWord(guess: String) -> Bool {
```

Comment:
This method doesn't use anything stored in the class, it probably belongs on something else, or should be a static method